bingads_downloader/downloader.py

```diff
@@ -48,9 +53,146 @@ def download_data_sets(api_client: BingReportClient):
     """
 
     authenticate_with_oauth(api_client)
+    download_account_structure_data(api_client)
     download_performance_data(api_client)
 
 
+def download_account_structure_data(api_client: BingReportClient):
+    """
+    Downloads the marketing structure for all accounts
+        Args:
+         api_client: BingAdsApiClient
+    """
+
+    filename = Path('bing-account-structure_{}.csv.gz'.format(config.output_file_version()))
```

Comment:
Why are we using format strings and two lines (line 69 )later literal string interpolation? I think we should aim for one way of doing things.